src/completers/completeTypes/README.md

```diff
@@ -1,20 +1,49 @@
 ## completeTypes - Completer
 
-This completer helps to extends a group of actions including their `SUCCESS` and `FAILURE` cases. It also receives a second parameter that describes which ones are not extended.
+This completer can extend the list of possible action types, helping to reduce its code size or generate them programatically. `completeTypes` receives an object with:
 
-Examples:
+* `primaryActions`: The completer generates the `_SUCCESS` and `_FAILURE` actions.
+* `ignoredActions`: The completer doesn't generate any extra actions for these.
+* `pollingActions`: The completer generates the `_SUCCESS`, `_FAILURE`, `_RETRY` and `_CANCEL` actions.
+* `customCompleters`: You can specify what types to generate.
+
```

Comment:
What's happened with `modalActions`?

Reply:
I'll just delete them, actually. They are not documented and never really used

Reply:
Same with `onSuscribe` and `onUnsuscribe` I think